topo_processor/stac/collection.py

```diff
@@ -62,6 +63,9 @@ def create_stac(self) -> pystac.Collection:
             href="./collection.json",
             license=self.license,
             providers=GLOBAL_PROVIDERS,
-            extent=pystac.Extent(pystac.SpatialExtent(bboxes=[0, 0, 0, 0]), pystac.TemporalExtent(intervals=[None, None])),
+            extent=pystac.Extent(
+                pystac.SpatialExtent(bboxes=[[0, 0, 0, 0]]),
+                pystac.TemporalExtent(intervals=[datetime.datetime.now(), datetime.datetime.now()]),
```

Comment:
The two values in the 'intervals' array might be different. Is that what you want?

Reply:
yes, the spatialExtent and TemporalExtent currently just placeholders to get our stac to pass validation. 
We need to create some new tickets to correctly sett these values.